pkg/controller/infrastructure/actuator_reconcile.go

```diff
@@ -92,16 +88,14 @@ func Reconcile(
 		return nil, nil, fmt.Errorf("could not create terraformer object: %+v", err)
 	}
 
+	terraformerInitializer, err := newInitializer(c, release, infrastructure)
```

Comment:
cc @ialidzhikov - can you double check these changes here?

Reply:
this code snippet does not exist anymore.